app/assets/javascripts/discourse/widgets/notification-item.js.es6

```diff
@@ -160,7 +160,12 @@ createWidget("notification-item", {
     let title;
 
     if (notificationName) {
-      title = I18n.t(`notifications.titles.${notificationName}`);
+      title =
+        notificationName === "custom"
```

Comment:
Nested ternaries are hard to understand. Can you re-write them using standard `if`/`else` for an easier reading?